config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "RapiDAST Configuration",
+    "description": "Schema for validating RapiDAST configuration files",
+    "type": "object",
+    "properties": {
+        "config": {
+            "type": "object",
+            "properties": {
+                "configVersion": {
+                    "default": 0,
+                    "$comment": "If no version is specified, the config converter will default to version 0",
+                    "type": "integer"
+                },
+                "base_results_dir": {
+                    "default": "./results",
+                    "type": "string"
+                },
+                "tls_verify_for_rapidast_downloads": {
+                    "default": true,
+                    "type": "boolean"
+                },
+                "environ": {
+                    "type": "object",
+                    "properties": {
+                        "envFile": {
+                            "type": "string"
+                        }
+                    },
+                    "required": []
+                },
+                "googleCloudStorage": {
+                    "type": "object",
+                    "properties": {
+                        "keyFile": {
+                            "type": "string"
+                        },
+                        "bucketName": {
+                            "default": "default-bucket-name",
+                            "type": "string"
+                        },
+                        "directory": {
+                            "type": "string"
+                        }
+                    },
+                    "comment": "Credentials are optional, as the client defaults to those inferred from the environment",
+                    "required": []
```

Comment:
bucketName can be required. WDYT?

Reply:
Yeah, some code with the default values may be unnecessary with this validation improvement. let's add bucketName to required.

Reply:
Added and also removed the default value from the code since it was never being used